modules/spotxBidAdapter.js

```diff
@@ -420,12 +421,9 @@ export const spec = {
   }
 }
 
-function createOutstreamScript(bid) {
+function createOutstreamScript(bid, script) {
   const slot = getBidIdParameter('slot', bid.renderer.config.outstream_options);
   logMessage('[SPOTX][renderer] Handle SpotX outstream renderer');
-  const script = window.document.createElement('script');
-  script.type = 'text/javascript';
-  script.src = 'https://js.spotx.tv/easi/v1/' + bid.channel_id + '.js';
```

Comment:
Instead of creating script in here, it is done by `loadExternalScript`

So it is passed into here after being inserted.